src/main/java/net/mcreator/ui/blockly/BlocklyPanel.java

```diff
@@ -223,12 +228,12 @@ public void removeLocalVariable(String name) {
 			return retval;
 
 		String[] vars = query.split(":");
-		for (String var : vars) {
-			String[] vardata = var.split(";");
+		for (String varName : vars) {
+			String[] vardata = varName.split(";");
 			if (vardata.length == 2) {
 				VariableElement element = new VariableElement();
 				element.setName(vardata[0]);
-				element.setType(BlocklyVariables.getMCreatorVariableTypeFromBlocklyVariableType(vardata[1]));
+				element.setType(Objects.requireNonNull(BlocklyVariables.getVarElementTypeFromBlocklyType(vardata[1])));

```

Comment:
It can happen plugin is removed and the variable no longer exists. This will then crash. Do null check and skip that variable (continue statement)